MapboxNavigationUI/MGLMapView+MGLNavigationAdditions.h

```diff
@@ -0,0 +1,7 @@
+#import <Mapbox/Mapbox.h>
+
+@interface MGLMapView (MGLNavigationAdditions)
+
+- (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations;
```

Comment:
As I mentioned in mapbox/mapbox-gl-native#6867, I think the map SDK should instead expose more specific methods like `-didUpdateLocationWithUserTrackingAnimated:`.